From: jammyjellyfish <jammyjellyfish255@outlook.com>
Date: Mon Apr 17 01:40:15 PM CST 2023
Subject: 支持指定 CC 编译
diff -up ipvsadm-1.31/libipvs/Makefile.orig2 ipvsadm-1.31/libipvs/Makefile
--- ipvsadm-1.31/libipvs/Makefile.orig2	2023-04-14 22:49:49.730489743 +0800
+++ ipvsadm-1.31/libipvs/Makefile	2023-04-14 22:50:04.302529121 +0800
@@ -1,6 +1,6 @@
 # Makefile for libipvs
 
-CC		= gcc
+CC		?= gcc
 CFLAGS		+= -Wall -Wunused -Wstrict-prototypes -g -fPIC
 ifneq (0,$(HAVE_NL))
 CFLAGS		+= -DLIBIPVS_USE_NL
diff -up ipvsadm-1.31/Makefile.orig2 ipvsadm-1.31/Makefile
--- ipvsadm-1.31/Makefile.orig2	2023-04-14 22:46:08.529892021 +0800
+++ ipvsadm-1.31/Makefile	2023-04-14 22:48:39.174299080 +0800
@@ -35,7 +35,7 @@ ARCH		= $(shell uname -m)
 RPMSOURCEDIR	= $(shell rpm --eval '%_sourcedir')
 RPMSPECDIR	= $(shell rpm --eval '%_specdir')
 
-CC		= gcc
+CC		?= gcc
 INCLUDE		=
 SBIN		= $(BUILD_ROOT)/sbin
 MANDIR		= usr/man
